### PR TITLE
Beta release notes 7.9

### PR DESCRIPTION
**Pull request by fpetru**
Add initial release notes for DietPi v.7.9 

- This includes also changes related to: https://github.com/MichaIng/DietPi-Docs/issues/622

### CODE REVIEW

**Comment by MichaIng** [comment]
I added an info about this being the last release with support for Debian Stretch: https://github.com/MichaIng/DietPi/commit/5aece88

**Comment by fpetru** [comment]
@MichaIng - I would be able to finish it early morning tomorrow. To not delay the things, please make the announcement on the GitHub as usual. Then, tomorrow, we publish an additional Tweet with these release notes too.

**Comment by MichaIng** [comment]
Many thanks. Yes this is fine, Twitter announcement is anyway planned for tomorrow early afternoon.

**Comment by fpetru** [comment]
@MichaIng @Joulinar - I know you wanted to add more items in the Beta release notes, highlighting some of the changes. If I missed any of them, please write a comment. Meanwhile, I'll fix the markdown issues. Thank you !

**Comment by MichaIng** [comment]
Since `#apache2` vs `#apache` is a fragment only, a redirect cannot be done. In theory the fragment can be "rewritten" but this would only have a server-side effect and does not change the browsers URL or its fragment, hence the scrolling wouldn't be fixed by that.

But it isn't such a big issue I think, users will see the top of the page, hence the TOC with Apache present.

File: docs/releases.md [hunk 1]
```diff
@@ -1,5 +1,97 @@
 # DietPi Releases
 
+## December 2021 (Beta Version 7.9) {: #december-2021-version-79 }
+
+### Overview
+
+**DietPi Beta v7.9 is ready for testing ! While we are in beta, we are working at full speed to roll out DietPi improvments and new features.**
```
Comment (MichaIng) [inline docs/releases.md:7]: Let's stick with no spaces before punctuation (`!`/`?`). Grammatically it is simply wrong, and I like to be consistent with this 🙂.

File: docs/releases.md [hunk 2]
```diff
@@ -1,5 +1,97 @@
...
+
+Take us for a spin, and let us know if you hit any road bumps along the way. The fun is in the journey.
+
+**Ready to get started ?** Find here the steps how to use the DietPi Beta branch [branch system](https://github.com/MichaIng/DietPi/blob/master/BRANCH_SYSTEM.md).
+
+### Announcement
+
+!!! Warning "**Debian "stretch" support**"
```
Comment (MichaIng) [inline docs/releases.md:15]: `stretch` => `Stretch`

File: docs/releases.md [hunk 3]
```diff
@@ -1,5 +1,97 @@
...
+
+    Debian 9.13 "stretch" was released July 18th, 2020, and it has been initially superseded by Debian 10 (buster), and then Debian 11 (bullseye).
```
Comment (MichaIng) [inline docs/releases.md:17]: `9.13` => `9`, some people may be on older point releases, but all of 9 is Stretch.

File: docs/releases.md [hunk 4]
```diff
@@ -1,5 +1,97 @@
...
+
+    **DietPi v.7.9 will be the last release with support for Debian stretch**. Next release will be **DietPi v8.0** and it will require Debian Buster or newer.
```
Comment (MichaIng) [inline docs/releases.md:19]: `v.7.9` => `v7.9`

File: docs/releases.md [hunk 5]
```diff
@@ -1,5 +1,97 @@
...
+        ![DietPi-Dashboard Default Port](assets/images/dietpi-dashboard-port-change.jpg){: width="800" height="252" loading="lazy"}
+
+        Many thanks to @blablazzz for reporting this issue: <https://github.com/MichaIng/DietPi/issues/4966>
+
+- [**DietPi-Backup**](../dietpi_tools/#dietpi-backup-backuprestore)
+    - A backup archive with a selectable amount of backups to keep can be created now. Backups are rotated automatically and if the maximum amount has been reached, the oldest backup is used as basis for the incremental new backup sync, to reduce writes and increase speed.
+
+        ![DietPi-Backup Amount](assets/images/dietpi-backup-amount.jpg){: width="800" height="265" loading="lazy"}
+
+        Many thanks to @johnvick and many others for requesting this feature [on the DietPi forum](https://dietpi.com/phpbb/viewtopic.php?t=3593).
```
Comment (MichaIng) [inline docs/releases.md:43]: Links to forum users need to be done explicitly:
```
@johnvick
```
=>
```
[phpBB:johnvick](https://dietpi.com/phpbb/memberlist.php?username=johnvick)
```

File: docs/releases.md [hunk 6]
```diff
@@ -1,5 +1,97 @@
...
+    - Default/base configuration is now added as separate file, so that the main `apache2.conf` is not touched anymore. Furthermore the default `vhost` is now pre-created before the package install, so that it can be skipped on a reinstall to not overwrite user customisations. These in combination allow for a safe and clean reinstall without breaking any changes done by the admin, with the little exception that the webroot is set to `/var/www` which is required for all our software options which make use of an external webserver.
+    - The new default configuration provides maximum privacy settings and security headers. It is simple to override these with own configurations, on `vhost` or directory level.
+    - Logging is now done to journal by default, and you can view them, running next command:
+
+        ```sh
+        journalctl -u apache2
+        ```
+
+    - The ServerName directive is added with the **local IP**, to mute related startup warnings.
```
Comment (MichaIng) [inline docs/releases.md:63]: "ServerName" should be wrapped into code tags `ServerName`.

File: docs/releases.md [hunk 7]
```diff
@@ -1,5 +1,97 @@
...
+- **DietPi-Software** :octicons-arrow-right-16: Resolved a DietPi v7.8 regression where [ReadyMedia](../software/media/#readymedia), [Deluge](../software/bittorrent/#deluge), [Sonarr](../software/bittorrent/#sonarr) and [Jellyfin](../software/media/#jellyfin) installs failed with an error on `usermod`, since the services were not stopped first. This has been loved via live patches for DietPi v7.8 as well.
+- **DietPi-Software** | **[Transmission](../software/bittorrent/#transmission)**: Resolved a v7.8 regression where on fresh installs the intended configuration was not deployed. Many thanks to @kannz and @alessandro.psrt for reporting these issue on the DietPi forum: ["Transmission settings ?"](https://dietpi.com/phpbb/viewtopic.php?t=9567) or ["Wrong settings.json in transmission-daemon"](https://dietpi.com/phpbb/viewtopic.php?t=9683).
+
+As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: <https://github.com/MichaIng/DietPi/issues/5019>
+
+### Known/Outstanding Issues {: #known-issues-79 }
```
Comment (MichaIng) [inline docs/releases.md:89]: We need to remove that heading from the old release notes then. IMHO it doesn't make sense to keep listing known issues which may have been resolved already, so I remove them from older ones and only keep it for the latest version.

File: docs/software/system_stats.md [hunk 8]
```diff
@@ -39,11 +39,15 @@ DietPi-Dashboard is a very lightweight and standalone web interface for monitori
...
-    DietPi-Dashboard is accessible by default at TCP port **8088**:
+    DietPi-Dashboard is accessible by default at TCP port **5252**:
 
-    - URL = `http://<your.IP>:8088`
+    - URL = `http://<your.IP>:5252`
     - Password = `<your software password>` (default: `dietpi`)
 
+    !!! warning "DietPi-Dashboard Port"
+
+        The default TCP network port has been changed from `8088` to `5252` to resolve a port conflict with InfluxDB. If you had an early installation of `dietpi-dashboard` and you did not want to make this change, you may continue to use the initial port (`8088`).
```
Comment (MichaIng) [inline docs/software/system_stats.md:49]: I think that warning it is not required here. Nothing is changed on existing installations without interactive confirmation, but all users who do the upgrade are informed and the choice offered.

File: docs/software/webserver_stack.md [hunk 9]
```diff
@@ -469,7 +469,7 @@ LLSP is a variation of the popular open source [LAMP web stack](#lamp-web-stack)
 
 ## WebServers
 
-### Apache2
+### Apache {: #apache }
```
Comment (MichaIng) [inline docs/software/webserver_stack.md:472]: `{: #apache }` is not required here, the heading text in lower case letters is used automatically.

**Comment by MichaIng** [comment]
Many thanks for also doing the "Apache2" => "Apache" renaming on all pages 👍. Good to have this aligned everywhere. I did some formal format/correction notes above. Please leave a not when starting to fix those, as I'll also start with this in an hour or so.

File: docs/releases.md (hunk 1 above)
Comment (fpetru) [reply docs/releases.md]: yes, fixed both

File: docs/software/system_stats.md (hunk 8 above)
Comment (fpetru) [reply docs/software/system_stats.md]: agree, transformed into an info field. I've even removed the header, so it's a subtle message.

**Comment by fpetru** [comment]
@MichaIng - resolved all reviewed notes

**Review by MichaIng — APPROVED** [review]

**Pull request merged**